pkg/cri/config/config.go

```diff
@@ -156,6 +156,9 @@ type Registry struct {
 	Auths map[string]AuthConfig `toml:"auths" json:"auths"`
 	// Headers adds additional HTTP headers that get sent to all registries
 	Headers map[string][]string `toml:"headers" json:"headers"`
+	// HostsDir sets the root directory for registry host configuration
+	// If a HostsDir is set, then registries other registry configs are ignored
```

Comment:
this could be read a number of ways.. though the code explains what is meant.

Reply:
I had a *really* hard time wording this :)